test/integration/model/bulk-create/include.test.js

```diff
@@ -621,6 +621,119 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         expect(persistedUsers[1].jobs).to.be.ok;
         expect(persistedUsers[1].jobs.length).to.equal(2);
       });
+
+      it('should pass options to associated models as well', async function () {
+        const Player = this.sequelize.define('Player', {
+          code: {
+            type: DataTypes.STRING,
+            primaryKey: true,
+          },
+          name: DataTypes.STRING,
+          surname: DataTypes.STRING,
+          dateBirth: DataTypes.DATE,
+          country: DataTypes.STRING,
+        });
+
+        const Team = this.sequelize.define('Team', {
+          code: {
+            type: DataTypes.STRING,
+            primaryKey: true,
+          },
+          name: DataTypes.STRING,
+          country: DataTypes.STRING,
+          color: DataTypes.STRING,
+          trophies: DataTypes.INTEGER,
+        });
+
+        Team.hasMany(Player, { as: 'players' });
+        Player.belongsTo(Team);
+
+        await this.sequelize.sync();
+
+        const dialect = this.sequelize.options.dialect;
+
+        if (dialect === 'mssql' || dialect === 'db2') {
+          expect(
+            async () => await Team.bulkCreate({}).to.throw(
```

Comment:
The updateOnDuplicate option is not used, how come this throws?

Reply:
Actually not sure. I can look into it in a separate PR - sorry about this line, had some stuff I had to take care of at home right as I was finishing and completely spaced here. I'll change to use the actual option this time 😅